src/features/pages/signup/form-container/signup-page-form.tsx

```diff
@@ -12,15 +12,16 @@ const SignUpPageForm = () => {
 
     const {
         register,
-        formState: { errors, isValid },
+        formState: { errors, isValid, isSubmitting, isSubmitSuccessful },
```

Comment:
```suggestion
        formState: { errors, is_valid, is_submitting, is_submit_successful },
```

Can we update this to follow our convention?

Reply:
Unfortunately these names are coming from react-hook-form library and we cannot rename them :(

Reply:
ok I see